Example/ios/ReanimatedExample.xcodeproj/project.pbxproj

```diff
@@ -583,6 +583,7 @@
 				"CODE_SIGN_IDENTITY[sdk=iphoneos*]" = "iPhone Developer";
 				COPY_PHASE_STRIP = NO;
 				ENABLE_STRICT_OBJC_MSGSEND = YES;
+				"EXCLUDED_ARCHS[sdk=iphonesimulator*]" = "arm64 ";
```

Comment:
I'm not sure it changes something 😄  but do you sure that it should be  `"arm64 "` instead of `"arm64"` (without space)

Reply:
It's not sth I did personally. I think it's a result of `pod install` cmd.